features/gba-dev/nationaliteit.feature

```diff
@@ -0,0 +1,243 @@
+  Dan heeft de response een persoon met een 'nationaliteit' met de volgende gegevens
+  | naam                       | waarde       |
+  | nationaliteit.code         | 0001         |
+  | nationaliteit.omschrijving | Nederlandse  |
+  | redenopname.code           | 000          |
+  | redenopname.omschrijving   | Onbekend     |
+  | datumIngangGeldigheid      | 19750707     |
+
+  Scenario: de persoon is Staatloos
+    Gegeven de persoon met burgerservicenummer '000000255' heeft een 'nationaliteit' met de volgende gegevens
+    | nationaliteit (05.10) | reden opnemen (63.10) | datum ingang geldigheid (85.10) |
+    | 0499                  | 312                   | 19750707                        |
+    Als gba personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000255                       |
+    | fields              | nationaliteiten                 |
+    Dan heeft de response een persoon met een 'nationaliteit' met de volgende gegevens
+    | naam                              | waarde                                           |
+    | nationaliteit.code                | 0499                                             |
+    | nationaliteit.omschrijving        | Staatloos                                        |
+    | redenOpname.code                  | 312                                              |
+    | redenOpname.omschrijving          | Vaststelling staatloosheid                       |
+    | datumIngangGeldigheid             | 19750707                                         |
+
+  Scenario: de persoon wordt behandeld als Nederlander, bijzonder Nederlanderschap heeft waarde "B"
+    Gegeven de persoon met burgerservicenummer '000000243' heeft een 'nationaliteit' met de volgende gegevens
+    | reden opnemen (63.10) | bijzonder Nederlanderschap (65.10) | datum ingang geldigheid (85.10) |
+    | 310                   | B                                  | 19570115                        |
+    Als gba personen wordt gezocht met de volgende parameters
+    | naam                | waarde                              |
+    | type                | RaadpleegMetBurgerservicenummer     |
+    | burgerservicenummer | 000000243                           |
+    | fields              | nationaliteiten                     |
+    Dan heeft de response een persoon met een 'nationaliteit' met de volgende gegevens
+    | naam                                | waarde                                  |
+    | aanduidingBijzonderNederlanderschap | B                                       |
+    | redenOpname.code                    | 310                                     |
+    | redenOpname.omschrijving            | Vaststelling bijzonder Nederlanderschap |
+    | datumIngangGeldigheid               | 19570115                                |
```

Comment:
wat B betekent wordt niet toegelicht. Is dat correct? Ik kan niet beoordelen of dat nodig is. Deze dev scenario's zijn misschien niet bedoeld om functioneel duidelijk te zijn.

Reply:
Zie het al, staat bij de end-tot-end features!